types/module/module.go

```diff
@@ -295,9 +295,6 @@ func (m *Manager) InitGenesis(ctx sdk.Context, cdc codec.JSONCodec, genesisData
 	var validatorUpdates []abci.ValidatorUpdate
 	ctx.Logger().Info("initializing blockchain state from genesis.json")
 	for _, moduleName := range m.OrderInitGenesis {
-		if genesisData[moduleName] == nil {
```

Comment:
for vesting module, the genesis data for this module is expected to be nil and it should initgenesis in this case